src/bindings/io/doc/CMakeLists.txt

```diff
@@ -0,0 +1,68 @@
+include(LibAddMacros)
+
+message(STATUS "Include Binding io_doc")
+
+set (BINDING_VARIANT doc)
+
+## Build library
+
+file (GLOB IO_VARIANT_HDR_FILES *.h)
+file (GLOB IO_VARIANT_SRC_FILES *.c)
```

Comment:
I am okay with GLOB (and we already use it often). I only wanted to mention that CMake guides discourage to do that because it won't track if you add new source files. (you'll need to invoke cmake manually.)

Reply:
I see. I removed it from I/O bindings (no benefit with one source and one header file) but left it for gathering the test sources - when there are more I/O bindings we can add a module with functions for repetitive parts (e.g. gathering test sources).